data/transactions/logic/assembler_test.go

```diff
@@ -410,7 +410,7 @@ const randomnessCompiled = "81ffff03d101d000"
 const v7Compiled = v6Compiled + "5e005f018120af060180070123456789abcd49490501988003012345494984" +
 	randomnessCompiled + "800243218001775c0280018881015d"
 
-const v8Compiled = v7Compiled + pairingCompiled + "8101e002fff800008101"
+const v8Compiled = v7Compiled + pairingCompiled + "81018a02fff800008101"
 
 var nonsense = map[uint64]string{
```

Comment:
@jannotti I have some minor feedback points that I expose via https://github.com/jannotti/go-algorand/pull/12.  Can you have a look?

Reply:
@jannotti Thanks for merging https://github.com/jannotti/go-algorand/pull/12.  I lack resolve permission, but let's consider the thread resolved.